docs/docs/ci.md

```diff
@@ -0,0 +1,138 @@
+Note that we do not use the upstream [PyPA manylinux images](https://github.com/pypa/manylinux/).
+Those do not contain CUDA, and attempting to maintain a fork of those images with CUDA proved quite challenging.
+Instead, we select an OS that we know has the glibc version associated with the manylinux standard that we wish to comply with, then choose [NVIDIA Docker images](https://hub.docker.com/r/nvidia/cuda) for that OS as the basis of our CI images.
+An appropriate OS by manylinux version may be found using [this handy website](https://github.com/mayeut/pep600_compliance).
+
+For wheels, separate images are created for build and testing.
+While the build images contain all libraries required for building RAPIDS, the test image is slimmed down and is missing some of these libraries.
+This allows us to test (at least to some extent) our manylinux compliance by verifying that our packages run even without those extra libraries installed.
+In an ideal world the testing image would not even have the CTK installed, but that is not currently possible because RAPIDS has some dependencies (numba and cupy) that are not as self-contained and still rely on a system CUDA installation.
+Until that changes, we cannot remove the CTK from wheel testing images.
+
+### conda images
+
+For conda, the images are different depending on whether they are for CUDA 11 or for CUDA 12+.
+As described in [the conda section above](packaging.md#conda) the available packages differed between these versions.
+Therefore, in RAPIDS conda CI images, for CUDA 11 images the CTK is installed on the system, while for CUDA 12 images it is not.
+The installation of conda itself in our CI images is handled upstream in the miniforge-cuda images (see below).
+
+The ci-imgs repository is automatically rebuild whenever new changes are pushed to any of the following:
+
+- dependency-file-generator
+- miniforge-cuda
+- gha-tools
+
+This ensures that the images always have the latest version of everything required by our CI build scripts.
+
+#### miniforge-cuda
+
+Our conda CI images are based on the [miniforge-cuda images](https://github.com/rapidsai/miniforge-cuda), which we also build.
+These images are based off of [`nvidia/cuda`](https://hub.docker.com/r/nvidia/cuda) Docker images, like the wheel images.
+They are a minimal installation of [`miniforge`](https://github.com/conda-forge/miniforge) on top of the CUDA images to enable conda in systems with the CTK installed.
+
+## Workflows
+
+### Runners
+
+RAPIDS runs Github actions on [a set of self-hosted runners at NVIDIA](https://docs.gha-runners.nvidia.com/).
```

Comment:
```suggestion
RAPIDS runs Github actions on a mixture of public and private runners[self-hosted runners at NVIDIA](https://docs.gha-runners.nvidia.com/).
```
We have public s3 runners still ( IIRC for arm )

Reply:
Yeah we definitely still run some jobs on public runners.